.github/workflows/bench-pr.yml

```diff
@@ -52,12 +52,28 @@ jobs:
         with:
           version: v1.2.1
 
-      - name: Run ${{ matrix.benchmark.name }} benchmark
+      - name: Build binary
```

Comment:
We build the binary before we start running the profiling agent, because we don't care about these traces